vertical-pod-autoscaler/e2e/v1/common.go

```diff
@@ -116,16 +128,33 @@ func SetupHamsterDeployment(f *framework.Framework, cpu, memory string, replicas
 	return d
 }
 
-// NewHamsterDeployment creates a simple hamster deployment for e2e test
-// purposes.
+// NewHamsterDeployment creates a simple hamster deployment for e2e test purposes.
 func NewHamsterDeployment(f *framework.Framework) *appsv1.Deployment {
```

Comment:
I think that it would be more straight forward when NewHamsterDeployment is just a call to NewNHamstersDeployment(f, 1)

Reply:
Done.